component/prometheus/remotewrite/remote_write_test.go

```diff
@@ -108,9 +82,146 @@ func Test(t *testing.T) {
 	}}
 
 	select {
-	case <-time.After(time.Minute):
+	case <-time.After(5 * time.Second):
+		require.FailNow(t, "timed out waiting for metrics")
+	case res := <-writeResult:
+		require.Equal(t, expect, res.Timeseries)
+	}
+}
+
+func TestUpdate(t *testing.T) {
```

Comment:
Added a test for update of `remote.write`. This was part of debugging to make sure some code paths do not panic when we Update components now - seems like we're safe. Worth keeping the test.